scripts/join_ceph_cluster.sh

```diff
@@ -78,10 +80,60 @@ for node in ncn-s001 ncn-s002 ncn-s003; do
   fi
 done
 
-# run preload images on host
-echo "Running pre-load-images on $host"
-if [[ ! $(/srv/cray/scripts/common/pre-load-images.sh) ]]; then
-  echo "ERROR  Unable to run pre-load-images.sh on $host."
+# run preload images on host only if Ceph version is not v16.2.13
+# if ceph is v16.2.13, then pull images from Nexus becuase
+# if ceph is v16.2.13, then local docker registry has been stopped
+version=""
+for node in ncn-s001 ncn-s002 ncn-s003; do
```

Comment:
double check the versions.  1.3 and 1.4 might have a minor version difference.

Reply:
In both CSM 1.3 and CSM 1.4, a Ceph upgrade is done to 16.2.13 in order for smartmon to be installed on storage nodes. See [CSM 1.3 patch docs](https://github.com/Cray-HPE/docs-csm/tree/release/1.3/upgrade/1.3.5#upgrade-ceph-and-stop-local-docker-registries) and [CSM 1.4 patch docs](https://github.com/Cray-HPE/docs-csm/tree/release/1.4/upgrade/1.4.3#upgrade-ceph-and-stop-local-docker-registries).